cmake/config.cmake

```diff
@@ -247,13 +235,13 @@ check_symbol_exists("SC_ENABLE_MPI" ${PROJECT_BINARY_DIR}/include/sc_config.h SC
 check_symbol_exists("SC_ENABLE_MPIIO" ${PROJECT_BINARY_DIR}/include/sc_config.h SC_ENABLE_MPIIO)
 
 # check consistency of MPI configuration
-if(mpi AND NOT (SC_ENABLE_MPI AND SC_ENABLE_MPIIO))
-  message(FATAL_ERROR "libsc MPI support was requested, but not configured in ${PROJECT_BINARY_DIR}/include/sc_config.h")
-endif()
-
-# check consistency of MPI I/O configuration
-if(mpi AND (SC_ENABLE_MPI AND NOT SC_ENABLE_MPIIO))
```

Comment:
It may be useful to still check if `SC_ENABLE_MPI` is activated but `SC_ENABLE_MPIIO` not, since the latter may be not available.

Reply:
We are checking this case explicitly to warn about the deprecated configuration setup.

Reply:
This is now checked in `check_mpiio.cmake`